subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -8,6 +8,7 @@ menuconfig NRF_CLOUD_PGPS
 	depends on MODEM_INFO
 	depends on MODEM_INFO_ADD_NETWORK
 	depends on DATE_TIME
+	depends on GNSS_MODULE
```

Comment:
The GNSS module is local to the asset tracker v2 application, while this Kconfig (`NRF_CLOUD_PGPS`) exists in the nRF Cloud P-GPS sub library. We shouldn't enforce a dependency to a application.

I think its fine the way it is right now for P-GPS. The only way P-GPS is enabled is by explicitly including `overlay-pgps.conf` in the west build command. So much like disabling the GNSS module the user has a choice to enable P-GPS.

The A-GPS changes are fine I think, becuase A-GPS is a default feature.